tests/functional/sync_v2_sign_up.js

```diff
@@ -23,7 +23,7 @@ define([
   var TOO_YOUNG_YEAR = new Date().getFullYear() - 13;
   var OLD_ENOUGH_YEAR = TOO_YOUNG_YEAR - 1;
 
-  var client;
+  var client; //eslint-disable-line no-unused-vars
```

Comment:
`client` doesn't seem to be used. Can we remove it and the lines here: https://github.com/mozilla/fxa-content-server/pull/2578/files#diff-420999c0bd8fd929634bff4b02e76788R67

We also no longer need the `FxaClient` module, so it can be removed from the module list: https://github.com/mozilla/fxa-content-server/pull/2578/files#diff-420999c0bd8fd929634bff4b02e76788R12, https://github.com/mozilla/fxa-content-server/pull/2578/files#diff-420999c0bd8fd929634bff4b02e76788R16